Mathlib/Algebra/Group/WithOne/Defs.lean

```diff
@@ -375,34 +377,12 @@ instance addMonoidWithOne [AddMonoidWithOne α] : AddMonoidWithOne (WithZero α)
           rw [Nat.cast_succ, coe_add, coe_one]
       }
 
-instance instLeftDistribClass [Mul α] [Add α] [LeftDistribClass α] :
-    LeftDistribClass (WithZero α) where
-  left_distrib a b c := by
-    cases' a with a; · rfl
-    cases' b with b <;> cases' c with c <;> try rfl
-    exact congr_arg some (left_distrib _ _ _)
-
-instance instRightDistribClass [Mul α] [Add α] [RightDistribClass α] :
-    RightDistribClass (WithZero α) where
-  right_distrib a b c := by
-    cases' c with c
-    · change (a + b) * 0 = a * 0 + b * 0
-      simp
-    cases' a with a <;> cases' b with b <;> try rfl
-    exact congr_arg some (right_distrib _ _ _)
-
-instance instDistrib [Distrib α] : Distrib (WithZero α) where
-  left_distrib := left_distrib
-  right_distrib := right_distrib
-
-instance semiring [Semiring α] : Semiring (WithZero α) :=
-  { WithZero.addMonoidWithOne, WithZero.addCommMonoid, WithZero.mulZeroClass,
-    WithZero.monoidWithZero, WithZero.instDistrib with }
-
 end WithZero
 
 -- Check that we haven't needed to import all the basic lemmas about groups,
 -- by asserting a random sample don't exist here:
 assert_not_exists inv_involutive
 assert_not_exists div_right_inj
 assert_not_exists pow_ite
+
```

Comment:
```suggestion
```

Reply:
This is on purpose: The comment above only applies to the three previous `assert_not_exists`, not to this one.